src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/TypeForwards.cs

```diff
@@ -2,14 +2,10 @@
 
 using System.Runtime.CompilerServices;
 
-#pragma warning disable RS0016 //  Type Forwarding may be removed in the future, when we accept breaking changes.
```

Comment:
Looks like this suppression is needed (or we add these to the API files).

Reply:
I think this is a case of VS analysis being different to the compiler, trying to see if updating the compiler fixes this.